features/test_feature.py

```diff
@@ -28,10 +28,10 @@
               'cub_device_scan_inclusive_sum', 'cub_device_scan_exclusive_sum', 'cub_device_select_unique', 'cub_device_radix_sort_keys', 'cub_device_radix_sort_pairs',
               'cub_device_select_flagged', 'cub_device_run_length_encide_encode', 'cub_counting_iterator', 'cub_arg_index_input_iterator',
               'cub_transform_iterator', 'activemask', 'complex', 'thrust-math'
-              'user_defined_rules', 'math-exec', 'math-ext-float', 'math-habs', 'math-ext-half', 'math-ext-half2', 'cudnn-activation',
+              'user_defined_rules', 'math-exec', 'math-habs', 'math-ext-half', 'math-ext-half2', 'cudnn-activation',
```

Comment:
From alloy test result: https://github.com/oneapi-src/SYCLomatic-test/actions/runs/3900237576/jobs/6660671847
math-ext-half, math-ext-half2 also met compile error:
```
math-ext-half COMPFAIL 
math-ext-half2 COMPFAIL 
```
Do them need to be removed?

Reply:
math-ext-half and math-ext-half2 failures are not compile issue. They are migration issue. It seems that the cuda API are not migrated according to the error message.